scripts/ci/images/ci_stop_arm_instance.sh

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Also not used.

Reply:
It is still referenced in https://github.com/apache/airflow/pull/45266/files#diff-30d32383eb4e00a8175f26f961aad8e18091fa6773df40ce981565ca652a9627R235 ?

Reply:
AARGH... Fixed :)